_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+images.  This was previously only possible when creating new images.  In
```

Comment:
'creating new disks'. Images are associated with the disk which contains an OS, right?

Reply:
I'm good with "disks".  thanks.